docs/zh/latest/admin-api.md

```diff
@@ -23,75 +23,76 @@ title: Admin API
 
 ## 目录
 
-* [Route](#route)
-* [Service](#service)
-* [Consumer](#consumer)
-* [Upstream](#upstream)
-* [SSL](#ssl)
-* [Global Rule](#global-rule)
-* [Plugin Config](#plugin-config)
-* [Plugin Metadata](#plugin-metadata)
-* [Plugin](#plugin)
+- [目录](#目录)
+- [Route](#route)
+- [Service](#service)
+- [Consumer](#consumer)
+- [Upstream](#upstream)
+- [SSL](#ssl)
+- [Global Rule](#global-rule)
+- [Plugin Config](#plugin-config)
+- [Plugin Metadata](#plugin-metadata)
+- [Plugin](#plugin)
 
 ## Route
 
-*地址*：/apisix/admin/routes/{id}?ttl=0
+_地址_：/apisix/admin/routes/{id}?ttl=0
```

Comment:
I am confusing by the style. Why sometimes you change `*blah*` to `_blah_` but sometimes you change `_blah_` to `*blah*`? And why should we change the style?

Reply:
just updated